config/prod.json

```diff
@@ -0,0 +1,37 @@
+{
+  "language": "Solidity",
+  "sources": {
+    "contracts/interfaces/IChainlinkOracle.sol": {
+      "urls": ["contracts/interfaces/IChainlinkOracle.sol"]
+    },
+    "contracts/interfaces/IChainlinkAggregatorV3.sol": {
+      "urls": ["contracts/interfaces/IChainlinkAggregatorV3.sol"]
+    },
```

Comment:
These can be removed as they aren't unique nor require entrypoints with respect to prod.